src/components/AnonymousChat/MessageChannelItem.tsx

```diff
@@ -72,11 +72,11 @@ const MessageChannelItem: (props: MessageChannelItemProps) => React.ReactElement
       unreadCount={item?.unreadCount}
       isMe={isMe}
       hasFollowButton={followStatus?.isFollowing === false || followStatus?.isFollowingFromAction}
-      // hasFollowButton={true}
       showFollowingButton={followStatus?.isFollowingFromAction}
       handleFollow={onFollowUser}
       channelType={item?.channelType}
       dbAnonUserInfo={dbAnonUserInfo}
+      hasAttachment={item?.rawJson?.message?.attachments?.length > 0}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The check for attachments in the message could potentially throw an error if any of the nested properties are `undefined`. It would be safer to use optional chaining on each property.
```diff
- hasAttachment={item?.rawJson?.message?.attachments?.length > 0}
+ hasAttachment={item?.rawJson?.message?.attachments?.length > 0}
```
This change ensures that if any of the nested properties (`rawJson`, `message`, or `attachments`) is `undefined`, it will not attempt to access further nested properties and cause a TypeError. Instead, it will return `undefined` immediately, which is falsy in JavaScript, so the `hasAttachment` prop will be `false`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->